packages/shared/src/components/modals/FeedFiltersModal/FeedFiltersIntroModal.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { ReactElement, useContext } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from '../StyledModal';
+import { ResponsiveModal } from '../ResponsiveModal';
+import UserIcon from '../../../../icons/user.svg';
+import { Button } from '../../buttons/Button';
+import FeedFiltersIntroModalTags from './FeedFiltersIntroModalTags';
+import { FeedFiltersIntroModalTagsContainer } from '../../utilities';
+import { Features, getFeatureValue } from '../../../lib/featureManagement';
+import FeaturesContext from '../../../contexts/FeaturesContext';
+import AnalyticsContext from '../../../contexts/AnalyticsContext';
+
+const tagsRows = [
+  ['', 'docker', '', 'kubernetes', ''],
+  ['', '', 'architecture', '', ''],
+  ['', '', '', 'devops', ''],
+  ['', 'cloud', '', '', ''],
+];
+
+const footerClass = {
+  introTest1: 'justify-center',
+  introTest2: 'justify-between',
+};
+
+type FeedFiltersProps = {
+  actionToOpenFeedFilters: () => unknown;
+  feedFilterModalType: string;
+};
+
+type ModalFooterProps = {
+  feedFilterOnboardingModalType: string;
+  onOpenFeedFilterModal: (event: React.MouseEvent<Element, MouseEvent>) => void;
+} & Pick<ModalProps, 'onRequestClose'>;
+
+type FeedFiltersIntroModalProps = FeedFiltersProps &
+  ModalFooterProps &
+  ModalProps;
+
+const IntroModalFooter = ({
+  feedFilterOnboardingModalType,
+  onRequestClose,
+  onOpenFeedFilterModal,
+}: ModalFooterProps) => {
+  const { trackEvent } = useContext(AnalyticsContext);
+  const closeModal = (event: React.MouseEvent) => {
+    trackEvent({
+      event_name: 'click',
+      target_type: 'skip button',
+      target_id: `intro modal ${feedFilterOnboardingModalType}`,
+      feed_item_title: 'skip',
+    });
+    onRequestClose(event);
+  };
+  const openFeedFilter = (event: React.MouseEvent) => {
+    trackEvent({
+      event_name: 'click',
+      target_type: 'open feed filter button',
+      target_id: `intro modal ${feedFilterOnboardingModalType}`,
+      feed_item_title:
+        feedFilterOnboardingModalType === 'introTest1'
+          ? 'Create my feed'
+          : 'Continue',
+    });
+    onOpenFeedFilterModal(event);
+  };
+  return (
+    <footer
+      className={classNames(
+        footerClass[feedFilterOnboardingModalType],
+        'flex fixed responsiveModalBreakpoint:sticky bottom-0 py-3 border-t border-theme-divider-tertiary bg-theme-bg-tertiary',
+      )}
+    >
+      {feedFilterOnboardingModalType === 'introTest2' && (
+        <Button
+          className="ml-4 w-20 text-theme-label-tertiary"
```

Comment:
I think it is inaccurate if we have to use a specific label color in our button. The text theme should be part of the button type you are using. I think you should use `btn-tertiary`

If you have any doubt, kindly check with the design team which type should actually be used.

![image](https://user-images.githubusercontent.com/13744167/166841546-a3f207df-a7cd-4aa1-8656-7439b91e7587.png)

```suggestion
          className="ml-4 w-20 btn-tertiary"
```